onnxruntime_extensions/_hf_cvt.py

```diff
@@ -126,25 +121,38 @@ def spm_decoder(self, **kwargs):
-                                     'BertDecoder',     HFTokenizerConverter.bpe_decoder, None),
-    "DistilBertTokenizer":
-                        TokenOpParam('BertTokenizer',   HFTokenizerConverter.bert_tokenizer,
-                                     'BertDecoder',     HFTokenizerConverter.bpe_decoder, None),
-    "GPT2Tokenizer":    TokenOpParam('GPT2Tokenizer',   HFTokenizerConverter.bpe_tokenizer,
-                                     'BpeDecoder',      HFTokenizerConverter.bpe_decoder, None),
-    "CLIPTokenizer":    TokenOpParam('CLIPTokenizer',   HFTokenizerConverter.clip_tokenizer,
-                                     'BpeDecoder',      HFTokenizerConverter.bpe_decoder, None),
-    "RobertaTokenizer": TokenOpParam("RobertaTokenizer",    HFTokenizerConverter.roberta_tokenizer,
-                                     None, None, None),
-    "T5Tokenizer":      TokenOpParam("SentencepieceTokenizer",  HFTokenizerConverter.spm_tokenizer,
-                                     "SentencepieceDecoder",    HFTokenizerConverter.spm_decoder,
-                                     default_inputs={'add_eos': [True]}),
-    "LlamaTokenizer":   TokenOpParam("SentencepieceTokenizer",  HFTokenizerConverter.spm_tokenizer,
-                                     "SentencepieceDecoder",    HFTokenizerConverter.spm_decoder,
-                                     default_inputs={'add_bos': [True]})
+    "BertTokenizer":        TokenOpParam('BertTokenizer',   HFTokenizerConverter.bert_tokenizer,
+                                         'BertDecoder',     HFTokenizerConverter.bpe_decoder, None),
+    "DistilBertTokenizer":  TokenOpParam('BertTokenizer',   HFTokenizerConverter.bert_tokenizer,
+                                         'BertDecoder',     HFTokenizerConverter.bpe_decoder, None),
+    "GPT2Tokenizer":        TokenOpParam('GPT2Tokenizer',   HFTokenizerConverter.bpe_tokenizer,
+                                         'BpeDecoder',      HFTokenizerConverter.bpe_decoder, None),
+    "CodeGenTokenizer":     TokenOpParam('GPT2Tokenizer',   HFTokenizerConverter.bpe_tokenizer,
+                                         'BpeDecoder',      HFTokenizerConverter.bpe_decoder, None),
+    "CLIPTokenizer":        TokenOpParam('CLIPTokenizer',   HFTokenizerConverter.clip_tokenizer,
+                                         'BpeDecoder',      HFTokenizerConverter.bpe_decoder, None),
+    "RobertaTokenizer":     TokenOpParam('RobertaTokenizer',        HFTokenizerConverter.roberta_tokenizer,
+                                         'BpeDecoder',              HFTokenizerConverter.bpe_decoder, None),
+    "BartTokenizer":        TokenOpParam('RobertaTokenizer',        HFTokenizerConverter.roberta_tokenizer,
+                                         'BpeDecoder',              HFTokenizerConverter.bpe_decoder, None),
+    "LayoutLMv3Tokenizer":  TokenOpParam('RobertaTokenizer',        HFTokenizerConverter.roberta_tokenizer,
```

Comment:
where did you find these tokenizers/did you look through the HF list and find ones using Roberta? trying to determine if there is a better way to know which ones are built on top ones we already have support for (like Roberta) so we can add more like this.

Reply:
check the huggingface source file which link was put in the comment, which can show you what the exact tokenizer type of one class is.